lib/apiservers/engine/backends/image.go

```diff
@@ -41,6 +43,7 @@ import (
 	"github.com/vmware/vic/lib/metadata"
 	"github.com/vmware/vic/lib/portlayer/util"
 	"github.com/vmware/vic/pkg/trace"
+	optrace "github.com/vmware/vic/pkg/trace"
```

Comment:
You already included trace above.  You're including a new one and aliasing it to optrace.